templates/repo/header.tmpl

```diff
@@ -14,7 +14,7 @@
 					<div class="gt-mx-2">/</div>
 					<a href="{{$.RepoLink}}">{{.Name}}</a>
 					{{if $.EnableFeed}}
-						<a href="{{$.RepoLink}}.rss"><i class="ui grey icon gt-ml-3" data-tooltip-content="{{$.locale.Tr "rss_feed"}}">{{svg "octicon-rss" 18}}</i></a>
+						<a class="muted gt-mx-2" href="{{$.RepoLink}}.rss" data-tooltip-content="{{$.locale.Tr "rss_feed"}}">{{svg "octicon-rss" 18}}</a>
```

Comment:
TBH, this is still not the right place for the `rss` link, because there could be following labels.

It doesn't bother me, so I can accept it.

Reply:
I agree, don't like it there either. Let me try something.

Reply:
Could have it in the repo buttons, but I'm not sure, it's a bit confusing there because some stuff there is for the branch:

<img width="516" alt="image" src="https://github.com/go-gitea/gitea/assets/115237/1410ec16-9747-46e8-8254-1c99f95c5e19">

Reply:
I'm undecided, will move after labels for now to match org page:

<img width="366" alt="image" src="https://github.com/go-gitea/gitea/assets/115237/79b02104-9e2c-4368-add0-a73fbd9d9a02">